src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Figure implements FigureProcess, FigureDrawing {
+    private String color;
+
+    public Figure() {
+
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: ";
+    }
+
+    @Override
+    public void draw() {
+
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }

```

Comment:
You don`t need this methods

Reply:
So, how to implement interfaces without methods?
![image](https://github.com/mate-academy/jv-oop-advanced/assets/71705897/02ab5564-52ae-4875-8b78-c520d6decf89)